### PR TITLE
issue #964, List active shifting records inside patient page

### PR DESCRIPTION
**Pull request by khavinshankar**
added active shifting record inside patient page

### CODE REVIEW

**Comment by khavinshankar** [comment]
The UI looks like this,
image

**Comment by dauntlessnomad** [comment]
looks good to merge

**Review by tomahawk-pilot — APPROVED** [review]

File: src/Components/Patient/PatientHome.tsx [hunk 1]
```diff
@@ -670,10 +715,178 @@ export const PatientHome = (props: any) => {
...
-          <div className="md:w-1/3 mx-2">
+          <div className={activeShiftingData.length ? "md:w-1/4 mx-2 h-full":"md:w-0 mx-2 h-full"}>
+          {
+            activeShiftingData.map((shift: any) => (
+              <div key={`shift_${shift.id}`}>
+                <div className="overflow-hidden shadow rounded-lg bg-white h-full">
+                  <div
+                    className={
+                      "p-4 h-full flex flex-col justify-between " +
+                      (shift.patient_object.disease_status == "POSITIVE"
```
Comment (riginoommen) [inline src/Components/Patient/PatientHome.tsx]:
```suggestion
                      (shift.patient_object.disease_status === "POSITIVE"
```

File: src/Components/Patient/PatientHome.tsx [hunk 2]
```diff
@@ -670,10 +715,178 @@ export const PatientHome = (props: any) => {
...
+                          </dt>
+                        </div>
+                        <div className="sm:col-span-1">
+                          <dt
+                            title=" Origin facility"
+                            className="text-sm leading-5 font-medium text-gray-500 flex items-center"
+                          >
+                            <i className="fas fa-plane-departure mr-2"></i>
+                            <dd className="font-bold text-sm leading-5 text-gray-900">
+                              {(shift.orgin_facility_object || {}).name}
```
Comment (riginoommen) [inline src/Components/Patient/PatientHome.tsx]:
```suggestion
                              {(shift.orgin_facility_object || {})?.name}
```

File: src/Components/Patient/PatientHome.tsx [hunk 3]
```diff
@@ -670,10 +715,178 @@ export const PatientHome = (props: any) => {
...
+                          </dt>
+                        </div>
+                        <div className="sm:col-span-1">
+                          <dt
+                            title="Shifting approving facility"
+                            className="text-sm leading-5 font-medium text-gray-500 flex items-center"
+                          >
+                            <i className="fas fa-user-check mr-2"></i>
+                            <dd className="font-bold text-sm leading-5 text-gray-900">
+                              {(shift.shifting_approving_facility_object || {}).name}
```
Comment (riginoommen) [inline src/Components/Patient/PatientHome.tsx]:
```suggestion
                              {(shift.shifting_approving_facility_object || {})?.name}
```

File: src/Components/Patient/PatientHome.tsx [hunk 4]
```diff
@@ -670,10 +715,178 @@ export const PatientHome = (props: any) => {
...
+                        </div>
+                        <div className="sm:col-span-1">
+                          <dt
+                            title=" Assigned facility"
+                            className="text-sm leading-5 font-medium text-gray-500 flex items-center"
+                          >
+                            <i className="fas fa-plane-arrival mr-2"></i>
+
+                            <dd className="font-bold text-sm leading-5 text-gray-900">
+                              {(shift.assigned_facility_object || {}).name ||
```
Comment (riginoommen) [inline src/Components/Patient/PatientHome.tsx]:
```suggestion
                              {(shift.assigned_facility_object || {})?.name ||
```

**Review by riginoommen — CHANGES_REQUESTED** [review]

**Comment by amit-kr-debug** [comment]
Looks like 
image

**Comment by khavinshankar** [comment]
@vigneshhari can you give it check

**Pull request merged**